### DISCUSSION TITLE
Apple Silicon M1 - add arm64e

### DISCUSSION THREAD

**Discussion by dimitre** [openframeworks/openFrameworks#7483] [Ideas]
I'm taking a look of what is needed to add arm64e to the build system and it seems apothecary scripts should be updated to work with libs. Apple M1 in fact use arm64e, so maybe we can only change arm64 to arm64e, what do you think?

I think the advantages are access to more modern intrinsics, maybe it means something to somebody (Advanced SIMD complex number support)
And maybe "Improved Javascript data type conversion support" can be useful too.

I've found about this difference trying to know what was happening with this warning
```
warning: ONLY_ACTIVE_ARCH=YES requested with multiple ARCHS and no active architecture could be computed;
```

cc: @2bbb @artificiel @ofTheo

**Comment by ofTheo** [comment]
if its as simple as arm64 -> arm64e for desktop that is totally fine. 

But if it means we have to do both for some older macOS's then I think we should wait on it. 

There is still quite a bit of more important things like: getting iOS to build simulator libs/xcframeworks properly, getting rpi4 64bit support, android etc

**Reply by dimitre** [reply]
ah great. I think it will work just by substitution, but I'll research a bit more.
